### DISCUSSION TITLE
could not load file or assembly 'cosmos.debug.kernel

### DISCUSSION THREAD

**Discussion by NuggetBoy12** [CosmosOS/Cosmos#2120] [General]
help when I debug the file I have this error: could not load file or assembly 'cosmos.debug.kernel, version=10.0.0.0, Culture=Neutral,
PublicKeyToken=5ae71220097cb983'. Does anyone know why I have this error?

**Comment by ghost** [comment]
I don't know this will helpful?
https://github.com/CosmosOS/Cosmos/issues/2114#issuecomment-1037136042
